apps/engineeringblog/app/layout.tsx

```diff
@@ -13,11 +14,12 @@ export const metadata: Metadata = {
     "Tech adventures in Africa's civic labs: Coding, innovating, and disrupting for good across the continent.",
 };
 
-export default function RootLayout({
+export default async function RootLayout({
```

Comment:
I was today years old to learn that in Next js components can return a promise @kilemensi.

Otherwise all good.